install.sh

```diff
@@ -260,6 +273,38 @@ set_synchrony_url() {
   fi
 }
 
+# Update the current load balancer listener on port 7999 to use the TCP protocol
+enable_tcp_protocol_on_lb_listener() {
+  local install_bitbucket
+  local region
+  local load_balancer_dns
+  local load_balancer_name
+  local original_instance_port
+
+  install_bitbucket=$(get_product "bitbucket" "${CONFIG_ABS_PATH}")
+
+  if [ -n "${install_bitbucket}" ]; then
+    region=$(get_variable 'region' "${CONFIG_ABS_PATH}")
+    load_balancer_dns=$(terraform output | grep '"load_balancer_hostname" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
+    load_balancer_name=$(echo "$load_balancer_dns" | cut -d '-' -f 1)
```

Comment:
Would you please change the address a variable in the script to the following pattern to be consistent:
```
"${variable}"
```